### PR TITLE
86b2j5p5v-venue-cards-poc: Venue cards with image carousel POC

### PR DESCRIPTION
**Pull request by LakshananS**
The carousel implementation in the provided code is a simple image slider component built using Next.js. It displays a series of images with smooth transitions between slides.

### CODE REVIEW

File: venue-reservation/src/Components/Carousel/page.tsx
Comment (bsaranga) [inline venue-reservation/src/Components/Carousel/page.tsx:1]: Let's use lower case directory and file names as a convention. Remove the folder named `Carousel` and rename `page.tsx` as `carousel.tsx` and rename `Component` to `component` and move `carousel.tsx` to `component.tsx`. Final dir structure should be `src/components/carousel.tsx`.

**Review by bsaranga — CHANGES_REQUESTED** [review]
Make requested changes.

File: venue-reservation/src/Components/Carousel/page.tsx
Comment (LakshananS) [reply venue-reservation/src/Components/Carousel/page.tsx:1]: I re-named and updated folders and file name.

**Comment by bsaranga** [comment]
@LakshananS Overall good, let's make the carousel component customizable by passing in props for container width, height, next/prev arrow colors, and dot indicator colors:

Props to be passed:
`width`
`height`
`arrow_bg_color`
`arrow_fg_color`
`dot_color`

**Comment by LakshananS** [comment]
* The Carousel component now accepts customizable props (width, height, arrowBgColor, arrowFgColor, dotColor, and activeDotColor).

* Each VenueCard can now pass different styles and sizes to the Carousel, making it reusable for multiple layouts and designs.

**Review by bsaranga — APPROVED** [review]

**Pull request merged**